docs/01-start-here/08-incidents.md

```diff
@@ -0,0 +1,115 @@
+# Incident Response & Triage
+
+what do when everything breaks
+
+The scope of this document is to define a process for dealing with Frontend incidents (site outages), and
+to prescribe a generic pattern of investigation tools and techniques for finding the cause of issues with
+the Frontend stack.
+
+## Sources of Truth
+
+All of our Frontend metrics, alerts and monitoring are available in two places: 
+
+* [CloudWatch](https://eu-west-1.console.aws.amazon.com/cloudwatch/) for health and performance metrics
+* [Kibana](https://logs.gutools.co.uk/app/kibana) for application logs
+
+You may also want to look at the [Fastly fashboard](https://manage.fastly.com/)
+
+## Process and roles
+
+Response to a potential site incidents will involve in general the following steps
```

Comment:
Hey thanks for writing this up. Can you include this doc which @nicl and I started as well? https://docs.google.com/document/d/1sAq378Oqm5NUG2_FJORDSd_Tag6gUUUsZaE9zUsgWHc/edit

Reply:
Sweet that means I can pull out some of the process parts and it'll make things a bit more focused here. Doc is still basically notes at this point will be updating for a while.

Reply:
Thanks @MatthewJWalls ! It is a great start. Very useful thing to be in our docs